NSSNamespace/User_Authentication
# Es product list view model

Details:

ProductList and ProductCreate view models now created.

To test:

Git fetch and checkout es-product-list-view-model. Ensure that product list view model is present in the Product View Models folder and contains an enumerable list of products. Ensure that product create view model is is in the same folder and contains a collection of select list items consisting of product type ids as well as a method for creating a new product saving it in the database via ApplicationDbContext.
